test/integration/grid-pro-custom-editor.test.js

```diff
@@ -280,4 +295,26 @@ describe('grid-pro custom editor', () => {
       expect(editor.value).to.equal('1984-01-13T00:00');
     });
   });
+
+  describe('custom-field', () => {
+    beforeEach(async () => {
+      grid = createGrid('expires');
+      await nextRender();
+      await editFirstCell();
+    });
+
+    it('should not stop editing when switching between fields using Tab', async () => {
```

Comment:
Could also add tests to verify that it's eventually possible to exit when Tab occurs on the last field.

Reply:
Done. Also added a test for switching fields inside `vaadin-custom-field` using mouse.